rust/gui-client/src-tauri/src/client/deep_link.rs

```diff
@@ -20,10 +20,11 @@ mod imp;
 #[path = "deep_link/windows.rs"]
 mod imp;
 
-// TODO: Replace this all for `anyhow`.
 #[cfg_attr(target_os = "linux", allow(dead_code))]
+#[cfg_attr(target_os = "macos", allow(dead_code))]
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
+    // This one is not `anyhow` since we catch it in the caller
```

Comment:
I only see that this is used to set the error message, you can use `anyhow` for that, for example set the message using `bail!` when the error occurs.

Reply:
That would work. The only reason I prefer this way is so that the string literal for a user-actionable error is closer to the dialog box code, and not down here.

For the WebviewNotInstalled one, in the future we could also offer to install it, but... it would be the distant future.